keras/layers/core/tf_op_layer.py

```diff
@@ -243,6 +243,9 @@ def _call_wrapper(*args, **kwargs):
     self._expects_training_arg = False
     self._expects_mask_arg = False
 
+  def compute_output_shape(self, input_shape=None):
+      return self.output_shape
```

Comment:
At what point is this property set?

Reply:
Thank you for your comments.
It's based on the fact that there is a `compute_output_shape `method  in the `convolution `layer,
`regularization `layer and so on, but `TFOpLambda `layer does not have this method.
Follow this link: https://colab.research.google.com/drive/1geUcvgluev88zRG-hGhRgJ4YbaA6pt2Y?usp=sharing,
When computing output shapes in `mobilenet_v3.MobileNetV3Large` which includes `TFOpLambda `layer , the error
suggests us to implement the `compute_output_shape` method on the `TFOpLambda `layer
But this error doesn't happen  in all other models in keras.applications because they do not have the `TFOpLambda `layer.